arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowTranslator.java

```diff
@@ -185,6 +187,36 @@ private String translateOp2(String op, String name, RexLiteral right) {
     return name + " " + op + " " + valueString + " " + valueType;
   }
 
+  /**
+   * Translates a call to a unary operator.
+   */
```

Comment:
Your javadoc format should be consistent

Reply:
The comment applies also to the other functions that were introduced

Reply:
Adjusted with [Adjust comment to follow the format of the other comments](https://github.com/apache/calcite/pull/3858/commits/3c48ffe1b54e1c2e1db5ff2a40089638bd67106a). I guess you meant putting it on one line?